runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -501,6 +512,31 @@ public void onGameTick(GameTick event)
 		}
 	}
 
+	@Subscribe
+	public void onGameStateChanged(GameStateChanged gameStateChanged)
+	{
+		// Check for game state changes which removes a teleblock (Hopping, Logging in or Login screen)
+		if (gameStateChanged.getGameState() == GameState.LOGGING_IN || gameStateChanged.getGameState() == GameState.HOPPING || gameStateChanged.getGameState() == GameState.LOGIN_SCREEN)
```

Comment:
Does this need to remove timers when logging in? Seems like removing them on `HOPPING` and `LOGIN_SCREEN` would be sufficient.